### PR TITLE
Day2

### PR DESCRIPTION
**Pull request by katrinkajb**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,16 +26,10 @@
                     What's your name?
                     <input type='text' name='name'>
                 </label>
```
Comment (katrinkajb) [inline index.html:28]: Pay attention to ____ and make sure you're doing xyz

**Review by katrinkajb — COMMENTED** [review]
Training exercise

File: index.html [hunk 2]
```diff
@@ -26,16 +26,10 @@
                     What's your name?
```
Comment (maddenlockin) [inline index.html:26]: what is YOUR name

**Review by maddenlockin — COMMENTED** [review]
whoop training review comments

File: quiz.data.js [hunk 3]
```diff
@@ -4,13 +4,11 @@ const colorQ = {
...
-        left: '44%'
+        top: '37%',
+        left: '55%'
     },
     image: '../assets/ice-kingdom.png',
-    description: `
-        Filler desc about needing scarves in the Ice Kingdom. Which color scarf do you choose?
-    `,
+    description: `The Ice King invites you to his castle, but it's awfully cold in here! You break out your trusty scarf and mittens to fight the chill. Which color scarf do you wear?`,
     choices: [{
```
Comment (tannermeck) [inline quiz.data.js:12]: Very concise data rendering for quiz information, looks great!

**Comment by trianacerda** [comment]
Great work!

**Pull request merged**